src/components/NavBar /ShoppingCartIcon.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { BiBookBookmark } from "react-icons/bi";
```

Comment:
try doing something like `import {BiBookBookmark as ShoppingCartIcon}` in /NavBar file instead of creating a new component.